.github/workflows/auto-assign.yml

```diff
@@ -0,0 +1,29 @@
+name: Auto-assign maintainer
+
+on:
+  issue_comment:
+    types:
+      - created
```

Comment:
We also need to support review comments. For example, this comment is a review comment, not an issue comment.